tests/profiling/collector/test_stack.py

```diff
@@ -455,7 +455,7 @@ def test_collect_span_id(tracer_and_collector):
         except IndexError:
             # No event left or no event yet
             continue
-        if span.trace_id == event.trace_id and span.span_id == event.span_id:
+        if span.span_id == event.span_id:
```

Comment:
do we need to assert the local root span id as well?

Reply:
I have a feeling the way this test is setup there is no local root span...

Reply:
Added assert for local root span as well